### ISSUE TITLE
[BUG] Nextjs: TypeError: Cannot read properties of undefined (reading '0')

### ISSUE THREAD

**Issue by nelwincatalogo** [family/connectkit#109]
**Dependencies:**
"connectkit": "^1.1.3",
"ethers": "5.7.2",
"wagmi": "^0.11.4"

```
// pages/_app.tsx
import { WagmiConfig, createClient, configureChains } from 'wagmi';
import { bscTestnet, bsc } from '@wagmi/core/chains';
import { ConnectKitProvider, ConnectKitButton } from 'connectkit';
import { publicProvider } from 'wagmi/providers/public';

const { chains, provider, webSocketProvider } = configureChains(
  [bscTestnet, bsc],
  [publicProvider()]
);

const client = createClient({
  autoConnect: false,
  provider,
  webSocketProvider,
});

const App = () => {
  return (
    <WagmiConfig client={client}>
      <ConnectKitProvider>
        /* Your App */
        <ConnectKitButton />
      </ConnectKitProvider>
    </WagmiConfig>
  );
};
```

![image](https://user-images.githubusercontent.com/45314780/216768737-dd2a81a8-7fe5-43bf-84be-a06f22855116.png)

**Comment by lochie** [comment]
@nelwincatalogo you may need to downgrade wagmi to `0.10.11` as we're still making an update for version `0.11.0` and above.
we're hoping to have an update out this week

**Comment by ryanberckmans** [comment]
I had this error, and had to downgrade to connectkit 1.1.1 to fix it. I currently have a working build of connectkit 1.1.1 and wagmi 0.11.5.

It seems this bug might have been introduced between connectkit 1.1.1 and 1.1.2. [Diff](https://github.com/family/connectkit/compare/v1.1.1...v1.1.2).

**Comment by duckdegen** [comment]
Any news on this issue?

**Comment by lochie** [comment]
Looks like this is a duplicate of https://github.com/family/connectkit/issues/133, but this should be resolved in ConnectKit version `1.2.1` with wagmi `0.11.x`.

If you're still seeing this issue please let us know. A reproducable codesandbox would also be incredibly helpful for us to debug further if necessary!
